.github/workflows/Update.yml

```diff
@@ -0,0 +1,53 @@
+name: Update
+
+on: [push]
+
+jobs:
+  Unit:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up PHP Version ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          tools: composer:v2
+          coverage: none
+
+      - name: Environment Check
+        run: |
+          php --version
+          composer --version
+
+      - name: "Determine tag"
+        id: "determine-tag"
+        run: "echo \"::set-output name=tag::${GITHUB_REF#refs/tags/}\""
+
+      - name: Get branch name (merge)
+        if: github.event_name != 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | tr / -)" >> $GITHUB_ENV
+
+      - name: Updaet Extension repo
+        run: |
+          mkdir -p ~/.ssh
+          openssl aes-256-cbc -K $encrypted_85bf36ae99bb_key -iv $encrypted_85bf36ae99bb_iv -in Resources/Private/deploy_rsa.enc -out ~/.ssh/id_rsa -d
+          echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ~/.ssh/config
+          chmod 600 ~/.ssh/id_rsa
+          git clone git@github.com:TYPO3-Console/Extension.git ../extension -b template
+          cd ../extension && git checkout origin/$BRANCH_NAME || true && cd -
+          composer extension-release
+          cd ../extension
+          git checkout README.md
+          git add .
+          if [ -n "${{ steps.determine-tag.outputs.tag }}" ]; then
+              git commit -m "Release extension version ${{ steps.determine-tag.outputs.tag }}" --author "Helmut Hummel <typo3@helhum.io>"
+              git tag ${{ steps.determine-tag.outputs.tag }}
+              git push --tags
+            else
+              git commit -m "Update extension based on commit $TRAVIS_COMMIT" --author "Helmut Hummel <typo3@helhum.io>"
```

Comment:
I'm not sure what this $TRAVIS_COMMIT value would be

Reply:
Is this what is ment? 


GITHUB_SHA | The commit SHA that triggered the workflow. For example, ffac537e6cbbf934b08745a378932722df287a53.
-- | --

From https://docs.github.com/en/actions/reference/environment-variables

Reply:
The travis docs says

> TRAVIS_COMMIT: The commit that the current build is testing.

Reply:
When looking at this commit it looks like it: 
https://github.com/TYPO3-Console/Extension/commit/5766481d7dfb028509d6a4c214f9b2b4fd6d9e62